migracja.rst

```diff
@@ -0,0 +1,235 @@
+danych w stosunku do danych zgromadzonych w systemie EZD. Na przykład, nie każdy dokument powstający
+w systemie EZD ma określonego adresata i co za tym idzie nie może to być element wymagany schematem XSD.
+W efekcie metadane pozbawione informacji o adresatach (odbiorcach) dokumentów będą automatycznie
+uznawane jako zgodne z rozporządzeniem 1519, gdyż paczka archiwalna będzie technicznie zgodna
+ze schematem XM. Mimo że takie dane nie będą zgodne z ogólnymi wytycznymi wynikającymi w art. 6
+ust. 1 i 1a ustawy o narodowym zasobie archiwalnym i archiwach (Dz.U.2016, poz.1506 ) gdyż paczka
+nie będzie zawierała kompletu danych odzwierciedlających sposób załatwiania sprawy, to nie da się
+tego zweryfikować automatycznie.
+
+Oprócz ustaleń dotyczących wymaganej kompletności danych, objaśnienia mogą być przydatne dla ustalenia
+przeznaczenia elementów zdefiniowanych schematem XML oraz dobrej praktyki ich wykorzystania
+do przekazania informacji w sposób odzwierciedlający przebieg załatwiania i rozstrzygania spraw,
+np. w celu wskazania w jaki sposób przekazywać ujednoliconą informację o dekretacjach i akceptacjach
+w systemach EZD, informacje o potwierdzeniu doręczenia itd. Brak takich informacji stanowiłby
+zubożenie w stosunku do papierowych akt spraw, w których dekretacje nanoszone wprost na dokumenty
+papierowe stanowią naturalną część dokumentacji.
+
+
+Wymagania interoperacyjne
+-------------------------
+
+Formaty danych oparte na składni XML są bardzo rozpowszechnione.
+Ze względu na fakt obowiązywania rozporządzenia 1519, które wprowadza format oparty na XML, oraz
+na fakt, że format proponowany do eksportu danych z EZD do sądów administracyjnych będzie
+prawdopodobnie oparty na składni XML i biorąc pod uwagę postulat podany na początku, by do wszystkich
+operacji masowego produkowania danych do wymiany, używany był syntaktycznie ten sam format danych — należy
+przyjąć, że format używany do migracji danych powinien używać składni XML i powinien mieć podobną budowę.
+
+Należy podkreślić, że bardzo ważnym jest, by formaty przeznaczone do
+
+- przekazywania paczki danych do innego EZD
+- przekazywania paczki danych do archiwum państwowego
+- przekazywania paczki danych do sądu
+- migracji danych
+
+były podzbiorami tego samego, spójnego, generalnego formatu wymiany danych.
+Spójność tych formatów znacznie ułatwi ich implementację, a zwłaszcza spowoduje duże ułatwienie
+w przyszłości, w sytuacjach gdy pojawi się potrzeba rozszerzenia zakresu przekazywanych danych.
+
+.. image:: images/zakres_danych_formatu_wymiany.svg
+
+Niestety, realizacja tego postulatu może być zagrożona przez fakt, iż prace nad formatem przekazywania
+danych dla sądów administracyjnych mają się zakończyć w marcu 2017 i mogą doprowadzić do powstania
+formatu który nie będzie opierał się o schemat XML ustalony w rozporządzeniu 1519 i, co za tym idzie,
+może być niezgodny technicznie z paczką archiwalną lub paczką danych do migracji.
```

Comment:
W jakim zakresie dziś możemy to ryzyko zminimalizować? Czy NSA jest otwarte na rozmowy w tym zakresie dziś? 

W moim zakresie możliwości wyjaśniania planów są ograniczone, gdyż NSA nie wykazuje żadnej [woli dzielenia się projektami dokumentów](https://files.jawne.info.pl/private_html/2016_12_a44b0270281bd0e51dc9a8d5cfb9564ec57f6bd9) wybierając łamanie prawa nawet kosztem [zanegowania swojego własnego orzecznictwa](http://informacjapubliczna.org.pl/11,905,trybunal_konstytucyjny_tez_musi_byc_transparentny.html), mimo doświadczeń co do konsekwencji braku przejrzystości życia publicznego np. przywołany kazus ustawy o Trybunale Konstytucyjnym stanowiącej podstawę do wielomiesięcznego kryzysu konstytucyjnego.